src/middle/Stmt.ml

```diff
@@ -0,0 +1,317 @@
+open Core_kernel
+open Common
+open Helpers
+
+(** Fixed-point of statements *)
+module Fixed = struct
+  module First = Expr.Fixed
```

Comment:
Sure - did you have something in mind? It will be used by both `Ast.Stmt` and `Mir.Stmt` eventually but calling it `Expr` seemed a little odd given the context.